analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -141,7 +135,7 @@ public static void LoopWithBreak(System.Collections.Generic.IEnumerable<string>
             {
                 if (condition)
                 {
-                    instance1.Dispose(); // FIXME Non-compliant
+                    instance1.Dispose(); // FIX ME - need to check the CFG as I'm not sure if this is an issue
```

Comment:
Leave it as FN. There's something tricky going on in too nested scenarios, not sure if it's this case too #7325